app.js

```diff
@@ -91,24 +92,20 @@ app.get(/\/(\d+)(?:x((\d+)))?(.\w+)?/, function (req, res, next) {
     })
 })
 
-// Start the server on the given port, or default to 9999
-var server = app.listen(process.env.PORT || 9999, function () {
+// Start the server on the given port, or default to 0
+server = app.listen(process.env.PORT || 9999, function () {
 
   // Output some useful info
-
-  var address = 'http://' + server.address().address + ':' + server.address().port
+  var url = 'http://' + ((server.address().address === '::') ? '[::]' : server.address().address) + server.address().port
 
   console.log(
-    [ ''
-    , '  Placeholder image server running at: ' + address
-    , ''
-    , '  Some example links to try:'
-    , '  -  ' + address + '/500x300'
-    , '  -  ' + address + '/500x300.jpg?text=Image!'
-    , '  -  ' + address + '/500x300.jpg?color=c00&textColor=fff'
-    , ''
-    , '  For more examples and usage, see the readme: ' + address
-    , ''
-    ].join('\n'))
+ `Placeholder image server running at: ${url}
```

Comment:
I know node <4 is not supported, but is it really worth the benefit to use backticks here and prevent anyone running on an older node?